protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -2282,6 +2284,28 @@ message ObserveDelegationsResponse {
 
 // -- Nodes --
 
+// Represents a basic node without any epoch specific details like delegations, staking, rewards etc.
+message NodeBasic {
```

Comment:
Is there a reason we can't use the normal `message Node` and just not fill in the epoch specific bits?

Reply:
Well I guess what epoch? The latest one?

Reply:
I am honestly not sure whether we need all the other stuff. Especially because this endpoint should return the entities changed by a tx hash.

Reply:
Hm, I though this might be more clear tbh. It might look like a bug when people never see the epoch specific bit.